.github/workflows/pr-review-requester.yml

```
@@ -1,7 +1,7 @@
 name: pr-review-requester
 
 on:
-  # pull_request_target is used to allow forks write permissions when running
+  # pull_request_target is used to allow forks to write permissions when running
```

not a typo. I think this should be reverted

```suggestion
  # pull_request_target is used to allow forks write permissions when running
```